narwhals/_duckdb/dataframe.py

```diff
@@ -321,3 +321,18 @@ def sort(
             )
         )
         return self._from_native_frame(result)
+
+    def drop_nulls(self: Self, subset: str | list[str] | None) -> Self:
+        import duckdb
+
+        rel = self._native_frame
+        subset_ = (
+            [subset]
+            if isinstance(subset, str)
```

Comment:
can we pack this at the `narwhals/dataframe.py` level, so that when we get here it's always `list[str] | None`?